packages/wdio-cli/src/utils.ts

```diff
@@ -840,177 +835,11 @@ export function detectPackageManager(argv = process.argv) {
     )) || 'npm'
 }
 
-/**
- * add ts-node if TypeScript is desired but not installed
- */
-export async function setupTypeScript(parsedAnswers: ParsedAnswers) {
-    /**
-     * don't create a `tsconfig.json` if user doesn't want to use TypeScript
-     */
-    if (!parsedAnswers.isUsingTypeScript) {
-        return
-    }
-
-    /**
-     * don't set up TypeScript if a `tsconfig.json` already exists but ensure we install `ts-node`
-     * as it is a requirement for running TypeScript tests
-     */
-    if (parsedAnswers.hasRootTSConfig) {
-        parsedAnswers.packagesToInstall.push('ts-node')
```

Comment:
I assume we also need to install `tsx` for the user if not installed already right?

Reply:
Yes, fixed by reinstating the whole function here